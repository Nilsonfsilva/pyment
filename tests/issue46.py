def func1(param1=[1, None, "hehe"]):
    pass


def func2(param1=(1, None, "hehe")):
    pass


def func3(param1={0: 1, "a": None}):
    pass

