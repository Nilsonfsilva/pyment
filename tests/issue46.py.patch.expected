# Patch generated by Pyment v0.3.2-dev4

--- a/issue46.py
+++ b/issue46.py
@@ -1,11 +1,26 @@
 def func1(param1=[1, None, "hehe"]):
+    """
+
+    :param param1:  (Default value = [1, None, "hehe"])
+
+    """
     pass
 
 
 def func2(param1=(1, None, "hehe")):
+    """
+
+    :param param1:  (Default value = (1, None, "hehe"))
+
+    """
     pass
 
 
 def func3(param1={0: 1, "a": None}):
+    """
+
+    :param param1:  (Default value = {0: 1, "a": None})
+
+    """
     pass
 
